extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -1,10 +1,3 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I had to axe comments locally because a copy/paste via `VI` in `bmo-extensions` would then comment every single line from then on down.  I'll add back before merge.